libsql-server/src/rpc/replica_proxy.rs

```diff
@@ -61,6 +63,7 @@ impl Proxy for ReplicaProxyService {
         &self,
         mut req: tonic::Request<ProgramReq>,
     ) -> Result<tonic::Response<ExecuteResults>, tonic::Status> {
+        tracing::debug!("execute");
```

Comment:
@LucioFranco are those going to be printed on every request? If so, that's `trace` level. Also, maybe some more context can be added here (and above, to the "stream_exec" message)?

Reply:
isn't it debug?

Reply:
They would be printed on each request if you specifically set the level to debug. We can later downgrade these to trace if we find they are too noisy, but for now they are 100% very helpful to understand when an rpc gets called.